src/oscar/templates/oscar/partials/form_field.html

```diff
@@ -22,12 +22,12 @@
             {% endblock %}
 
             {% block controls %}
-                <div class="{% if style|default:"stacked" != 'stacked' %}col-sm-7{% endif %}">
+                <div class="{% if style|default:"stacked" != 'stacked' %}col-sm-7{% endif %}{% if field.widget_type == 'CheckboxInput' %} checkbox{% endif %}">
```

Comment:
We are applying the `checkbox` class to both the column and to the label, which I don't think is a good idea. I think this should be controlled at the field level exclusively.